class/Defaults/Trigger/WordPress/DeactivatedPlugin.php

```diff
@@ -0,0 +1,103 @@
+<?php
+/**
+ * WordPress Deactivate Plugin trigger
```

Comment:
Better grammar needed, ie. `WordPress plugin deactivated trigger` that shows an action being just done